content/guides/references/troubleshooting.md

```diff
@@ -630,42 +630,3 @@ workaround to the relevant Cypress GitHub issue. It will help us release an
 official fix faster.
 
 </Alert>
-
-### Edit the installed Cypress code
-
-The installed application comes with the fully transpiled, unobfuscated
-JavaScript source code that you can hack on. You might want to directly modify
-the installed app code to:
-
-- investigate a hard to recreate bug that happens on your machine
-- change the run-time behavior of Cypress before opening a pull request
-- have fun 🎉
-
-First, print where the binary is installed using the
-[cypress cache path](/guides/guides/command-line#cypress-cache-path) command.
-
-For example, on a Mac:
-
-```shell
-npx cypress cache path
-/Users/jane/Library/Caches/Cypress
-```
-
-Second, open the source code at the following path in any code editor. Make sure
-to substitute `3.3.1` for the desired version of Cypress you want to edit.
-
-```text
-/Users/jane/Library/Caches/Cypress/3.3.1/Cypress.app/Contents/Resources/app/packages/
-```
-
-You can change anything in the JavaScript code:
-
-<DocsImage src="/img/guides/troubleshooting/source-code.png" alt="Source code of Cypress in a text editor" ></DocsImage>
```

Comment:
I think we can remove this image as well. It doesn't seem to be referenced anywhere else.

Reply:
Good catch.

Reply:
Removed: [`92e4ed8` (#4825)](https://github.com/cypress-io/cypress-documentation/pull/4825/commits/92e4ed8b44eecee78751c7cad748a1ef8814f626)